ksql-clickstream-demo/non-docker-clickstream.md

```diff
@@ -247,11 +247,11 @@ These steps will guide you through how to setup your environment and run the cli
     Your output should resemble:
 
     ```bash
-    1503585475000 | 4 : Window{start=1503585475000 end=-} | 4 | 14
-    1503585480000 | 25 : Window{start=1503585480000 end=-} | 25 | 9
-    1503585480000 | 16 : Window{start=1503585480000 end=-} | 16 | 6
-    1503585475000 | 25 : Window{start=1503585475000 end=-} | 25 | 20
-    1503585480000 | 37 : Window{start=1503585480000 end=-} | 37 | 6
+    1503585475000 | 1503585475000 | 1503585480000 | 4 | 4 | 14
+    1503585480000 | 1503585480000 | 1503585485000 | 25 | 25 | 9
+    1503585480000 | 1503585480000 | 1503585485000 | 16 | 16 | 6
+    1503585475000 | 1503585475000 | 1503585480000 | 25 | 25 | 20
+    1503585480000 | 1503585480000 | 1503585485000 | 37 | 37 | 6
```

Comment:
Updated to reflect changes in `ROWKEY` and addition of `WINDOWSTART` and `WINDOWEND`.

QQ: is this file used/needed? should this file be removed?  all the other docs for the clickstream demo are now elsewhere, as is the code. cc @JimGalasyn @rmoff

Reply:
You're right, any non-Docker topics don't belong in the new doc set (for now), but I've kept them around for future use, which of course is confusing.